client/go/outline/callback/callback.go

```diff
@@ -0,0 +1,97 @@
+// Copyright 2025 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package callback provides a thread-safe mechanism for managing and invoking callbacks.
+package callback
+
+import (
+	"fmt"
+	"log/slog"
+	"sync"
+)
+
+// Token can be used to uniquely identify a registered callback.
+type Token string
```

Comment:
Why don't you use the id as the token? In fact, you can simplify and remove the token concept. It's simply a callbackId

Reply:
Done.